_data/toc.yaml

```diff
@@ -595,6 +595,14 @@ reference:
         title: docker scout quickview
       - path: /engine/reference/commandline/scout_recommendations/
         title: docker scout recommendations
+      - path: /engine/reference/commandline/scout_repo/
```

Comment:
@thaJeztah I can see some inconsistency in the docs where we sometimes just saw the top level command and other times all the sub commands. Any preference?